webpack.config.js

```diff
@@ -276,7 +274,7 @@ if ( shouldMinify ) {
 			cache: true,
 			parallel: true,
 			uglifyOptions: { ecma: 5 },
-			sourceMap: false,
+			sourceMap: Boolean( process.env.SOURCEMAP ),
```

Comment:
is this right? does uglify just need a boolean or does it too take a string? I didn't question it actually

Reply:
[uglify plugin docs](https://github.com/webpack-contrib/uglifyjs-webpack-plugin#options) indicate it only expects a bool

Reply:
Yeah, it actually crashed when given a string value.